tests/test_clan_members_rank.py

```diff
@@ -47,12 +48,14 @@ def test_compare_card_levels(self):
 
     def test_get_card_level_counts(self):

```

Comment:
You should also add a test that just test's `get_card_level_counts` similar to what I've done but with a filter added too